mongodbatlas/clusters.go

```diff
@@ -30,8 +30,9 @@ var _ ClustersService = &ClustersServiceOp{}
 
 // AutoScaling configures your cluster to automatically scale its storage
 type AutoScaling struct {
-	DiskGBEnabled *bool    `json:"diskGBEnabled,omitempty"`
-	Compute       *Compute `json:"compute,omitempty"`
+	AutoIndexingEnabled *bool    `json:"autoIndexingEnabled,omitempty"` // Autopilot mode is only available if you are enrolled in the Auto Pilot Early Access program.
```

Comment:
Good add, we'll need to support this in Terraform post beta.

Reply:
I did a quick pass to see if there was anything else new we were missing and this was the only other thing

Reply:
Thank you for doing that!